Docs/ROADMAP.md

```diff
@@ -0,0 +1,328 @@
+it might very well be the case that I will chip in a couple hundred dollars in order to get some ads on websites once our product is polished.
+
+### Epics
+
+#### Characters
+
+We will need rigged models, textures, animations, and animation controllers, and voice-overs for all characters in this game.
+This should cover all three playable characters as well as a number of enemy characters.
+
+Each character should have a unique play style or niche in combat. This can partly be provided by a specific set of ICEPAWS stats, but should be reinforced with a set of 
+abilities that can be acquired and upgraded to modify this play style or niche.
+
+#### Items/abilities/upgrades
+
+We will need a set of items and abilities that can be used with all humanoid characters. If we have non-humanoid characters, we'll need abilities(ie biting, flying) that 
+may be unique to them.
+
+Each item or ability should have at least one possible upgrade that changes how it functions. These can come in the form of simply damage increases, or can introduce 
+entirely new functionality. (ie a crossbow that does less damage, but knocks the enemy back very far).
+
+#### Menus
+
+There are many menus outlined above in this document. We will need to build out and polish these menus where applicable, in order to deliver an intuitive and visually 
+appealing experience for the user.
+
+#### Music
+
+We will need a soundtrack that fits the game's mood and tone. We can use any music released CC-BY, or develop it outselves.
+This music will chiefly be broken into the following categories:
+- Opening menu music
+- Combat music
+- Victory music
+- Defeat music
+
+#### Sound effects
+
+We will need sound effects for just about every in-game action as well as audio cues when a player navigates through menus or
+earns points/experience/currency.
+
+#### Level design
+
+We will need to design levels for the arena encounters. All scenery and terrain will need to be modeled and textured.
+Each level should contain spawn points for players, enemies, and powerups. Interactive elements such as doors or destructable
+windows will also need to be developed in order to give
+
+#### User experience/balancing
+
+It will be critical to focus on delivering a good experience to our end users. To do this, we will need to allow external playtesting and respond
+to feedback provided. If a feature does not positively contribute to the user's experience, it should be modified or removed.
+
+One common point of contention is balancing the rewards for different play styles so that they all feel valid to the player. This is highly subjective and will require much consideration.
+
+#### Writing
+
+Any dialogue, copy, or decisions regarding lore/story will need to be made as cohesively and consistently as possible. The game should have a consistent set of themes that 
+it adheres to.
+
+#### 2D Art
+
+Many UI elements will require us to make images.
```

Comment:
This might be covered under all the respective topics/is more like a component of the other epics versus an epic itself I think?